dlt/destinations/impl/clickhouse/sql_client.py

```diff
@@ -217,3 +217,9 @@ def _make_database_exception(cls, ex: Exception) -> Exception:
     @staticmethod
     def is_dbapi_exception(ex: Exception) -> bool:
         return isinstance(ex, clickhouse_driver.dbapi.Error)
+
+    def _get_deployment_type(self) -> TDeployment:
```

Comment:
Seems this is only used in the tests. Probably better to turn it into a test utility.

(I did the same in a PR once, and that was the feedback I received then.)

Reply:
Agreed - refactored into test utils